lib/Rex/Config.pm

```diff
@@ -1762,7 +1766,11 @@ sub read_ssh_config_file {
   $config_file ||= _home_dir() . '/.ssh/config';
 
   if ( -f $config_file ) {
-    my @lines = eval { local (@ARGV) = ($config_file); <>; };
+    my @lines = do {
+      use IO::File;
+      my $fh = IO::File->new( $config_file, 'r' );
+      join '', $fh->getlines();
```

Comment:
Can it be that `t/config-ssh.t` fails because now all lines of the file is concatenated with `join` while it was treated as an array of lines before?